src/twisted/python/_pydoctor.py

```diff
@@ -114,9 +115,13 @@ def _getURLAsHEAD(self, url):
         @type url: L{str}
 
         @return: The response for the HEAD method.
-        @rtype: urllib2 response
+        @rtype: urllib2 response or L{None}
         """
-        return urllib2.urlopen(HeadRequest(url))
+        try:
+            return urllib2.urlopen(HeadRequest(url))
+        except Exception as e:
+            print("Error opening {}: {}".format(url, e))
```

Comment:
https://github.com/twisted/twisted/pull/694/files#diff-aa8f1554609c110cd5bedee40ed3e19dR37 says there's a workflow for dealing with such things.  Can you do what that docstring says or fix the docstring if it says to do the wrong thing?